packages/discount-code-importer/readme.md

```diff
@@ -1,6 +1,6 @@
 # discount-code-importer
 
-Import discount codes to the commercetools platform.
+Import [commercetools discount codes](https://dev.commercetools.com/http-api-projects-discountCodes.html) to the commercetools platform.
 
 https://commercetools.github.io/nodejs/api/discount-code-importer/
```

Comment:
Is this URL correct? For example `csv-parser-price` is on this URL:
https://commercetools.github.io/nodejs/cli/csv-parser-price.html
So I would expect that this doc will be here:
```
https://commercetools.github.io/nodejs/cli/discount-code-importer.html
https://commercetools.github.io/nodejs/api/discount-code-importer/ <-- instead of this
```